src/main/java/co/inventorsoft/academy/service/FileWriterService.java

```diff
@@ -15,8 +14,7 @@ public class FileWriterService {
      */
     public void write(String str) {
         try {
```

Comment:
What will happen if we will have an exception on line 18? BufferedWriter will be closed or no? Try to use try-with-resources or finally block.